Dragonflight/ShamanRestoration.lua

```diff
@@ -766,6 +791,40 @@ spec:RegisterAbilities( {
             applyBuff( "stormkeeper" )
         end,
     },
+	
+	-- Talent: Resets the cooldown of your most recently used totem with a base cooldown shorter than 3 minutes.
+    totemic_recall = {
```

Comment:
Totemic Recall exists in Elemental and Enhancement modules.  If it's being brought over, the special mechanics related to `recall_totem_1` and `recall_totem_2` should be brought over as well.

Reply:
Okay, that sounds reasonable, even if no one takes that Creation Core talent for now, anything can change in the future.